Ui/Component/Listing/Column/Carrier.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Show send date column
+ *
+ * If you want to add improvements, please create a fork in our GitHub:
+ * https://github.com/myparcelbe
+ *
+ * @author      Reindert Vetter <info@sendmyparcel.be>
+ * @copyright   2010-2017 MyParcel
+ * @license     http://creativecommons.org/licenses/by-nc-nd/3.0/nl/deed.en_US  CC BY-NC-ND 3.0 NL
+ * @link        https://github.com/myparcelbe/magento
+ * @since       File available since Release v0.1.0
```

Comment:
```suggestion
 * @since       File available since Release v3.0.0
```

Reply:
Ben ik niet met je eens, in de eerdere versies bestond dit bestand ook al.

Reply:
Done